src/datareaders/data_object_holders.py

```diff
@@ -0,0 +1,38 @@
+class PointType:
+
+    def __init__(self, name, return_type, units=None, factor=None, enumeration_values=None):
+        self.name = name
+        self.return_type = return_type  # Am I an int, a float, or enumerated?
+        self.units = units  # String, might stay undefined if not a numerical point
+        self.factor = factor  # Integer, might stay undefined if not a numerical point
+        self.enumeration_values = enumeration_values   # might stay undefined if not an enumerated point
```

Comment:
So the enum values are actually stored in the units column right? But you split them out here so that you can join them with a comma for the placeholder right? Just trying to wrap my head around this

Reply:
Yeah, in the table they go in the units column because enumerated types don't have units, and it was just a convenient string column to put them it. In this class I split them out so that I could think about them separately and so that siemens_reader.py can use the list rather than the comma joined string (ignore the different namings between enumeration_values and enumeration_settings, I'm fixing that). I welcome any suggestions if you think there would be a better way to store them either in the code or the database.

Reply:
Makes sense to me